deps/oblib/src/lib/CMakeLists.txt

```diff
@@ -397,6 +397,7 @@ ob_lib_add_pchs(lib
   compress/snappy/ob_snappy_compressor.h
   compress/none/ob_none_compressor.h
   compress/ob_compress_util.h
+  compress/zip/ob_zip_compressor.h
```

Comment:
Do not add uncommon files to the PCH list. This will affect compilation efficiency.

Reply:
ok